crates/utils/src/error.rs

```diff
@@ -165,6 +165,8 @@ pub enum LemmyErrorType {
   InvalidUnixTime,
   InvalidBotAction,
   CantBlockLocalInstance,
+  TaglineContentRequired,
+  TaglineContentLengthOverflow,
```

Comment:
Just use a single error `TaglineInvalid`. Not worth making separate errors which have to be translated into dozens of languages.